### PR TITLE
fix whatsapp link bug

### PR DESCRIPTION
**Pull request by mehul-gohappy**
remaining fixxess

### CODE REVIEW

File: components/Profile.js [hunk 1]
```diff
@@ -163,15 +166,19 @@ class Profile extends Component {
       if (response.data) {
         const properties = response.data.properties;
         if (properties && properties.length > 0) {
+          console.log(properties);
```
Comment (Rakshit1411) [inline components/Profile.js:169]: remove console.log

File: screens/homeScreen/HomeScreen.js [hunk 2]
```diff
@@ -79,22 +79,10 @@ class HomeScreen extends Component {
...
           {this.props.profile.age != null ? (
             this.props.profile.age > 50 && (
-              <WhatsAppFAB
-                url={
-                  this.props.profile.properties
-                    ? this.props.profile.properties.whatsappLink
-                    : this.state.whatsappLink
-                }
-              />
+              <WhatsAppFAB/>
```
Comment (Rakshit1411) [inline screens/homeScreen/HomeScreen.js:82]: what is the diff between these 2 ?

File: screens/overview/OverviewScreen.js [hunk 3]
```diff
@@ -170,23 +170,9 @@ class OverviewScreen extends Component {
...
-            this.props.profile.age > 50 && (
-              <WhatsAppFAB
-                url={
-                  this.props.profile.properties
-                    ? this.props.profile.properties.whatsappLink
-                    : this.state.whatsappLink
-                }
-              />
-            )
+            this.props.profile.age > 50 && <WhatsAppFAB />
```
Comment (Rakshit1411) [inline screens/overview/OverviewScreen.js:173]: what is the diff between 2 ?

**Review by Rakshit1411 — CHANGES_REQUESTED** [review]

File: screens/homeScreen/HomeScreen.js (hunk 2 above)
Comment (mehul-gohappy) [reply screens/homeScreen/HomeScreen.js:82]: we are fetching the whatsapp link in the WhatsAppFAB component when it mounts so we don't need to pass any url explicitly

File: screens/overview/OverviewScreen.js (hunk 3 above)
Comment (mehul-gohappy) [reply screens/overview/OverviewScreen.js:173]: we are fetching the whatsapp link in the WhatsAppFAB component when it mounts so we don't need to pass any url explicitly

**Review by Rakshit1411 — APPROVED** [review]

**Pull request merged**